Add NodeJS in the base Docker image

```
1590
INFO: Hit the cache for 0 out of 0
1591
ERROR: Error when running: 'node -v'. Is Node.js available during analysis?
1592
org.sonarsource.nodejs.NodeCommandException: Error when running: 'node -v'. Is Node.js available during analysis?
1593
	at org.sonarsource.nodejs.NodeCommand.start(NodeCommand.java:79)
1594
	at org.sonarsource.nodejs.NodeCommandBuilderImpl.getVersion(NodeCommandBuilderImpl.java:203)
1595
	at org.sonarsource.nodejs.NodeCommandBuilderImpl.checkNodeCompatibility(NodeCommandBuilderImpl.java:169)
1596
	at org.sonarsource.nodejs.NodeCommandBuilderImpl.build(NodeCommandBuilderImpl.java:143)
1597
	at org.sonar.plugins.javascript.eslint.EslintBridgeServerImpl.initNodeCommand(EslintBridgeServerImpl.java:201)
1598
	at org.sonar.plugins.javascript.eslint.EslintBridgeServerImpl.startServer(EslintBridgeServerImpl.java:142)
1599
	at org.sonar.plugins.javascript.eslint.EslintBridgeServerImpl.startServerLazily(EslintBridgeServerImpl.java:233)
1600
	at org.sonar.plugins.javascript.eslint.AbstractEslintSensor.execute(AbstractEslintSensor.java:68)
1601
	at org.sonar.scanner.sensor.AbstractSensorWrapper.analyse(AbstractSensorWrapper.java:64)
1602
	at org.sonar.scanner.sensor.ModuleSensorsExecutor.execute(ModuleSensorsExecutor.java:88)
1603
	at org.sonar.scanner.sensor.ModuleSensorsExecutor.lambda$execute$1(ModuleSensorsExecutor.java:61)
1604
	at org.sonar.scanner.sensor.ModuleSensorsExecutor.withModuleStrategy(ModuleSensorsExecutor.java:79)
1605
	at org.sonar.scanner.sensor.ModuleSensorsExecutor.execute(ModuleSensorsExecutor.java:61)
1606
	at org.sonar.scanner.scan.SpringModuleScanContainer.doAfterStart(SpringModuleScanContainer.java:82)
1607
	at org.sonar.core.platform.SpringComponentContainer.startComponents(SpringComponentContainer.java:188)
1608
	at org.sonar.core.platform.SpringComponentContainer.execute(SpringComponentContainer.java:167)
1609
	at org.sonar.scanner.scan.SpringProjectScanContainer.scan(SpringProjectScanContainer.java:403)
1610
	at org.sonar.scanner.scan.SpringProjectScanContainer.scanRecursively(SpringProjectScanContainer.java:399)
1611
	at org.sonar.scanner.scan.SpringProjectScanContainer.scanRecursively(SpringProjectScanContainer.java:396)
1612
	at org.sonar.scanner.scan.SpringProjectScanContainer.doAfterStart(SpringProjectScanContainer.java:368)
1613
	at org.sonar.core.platform.SpringComponentContainer.startComponents(SpringComponentContainer.java:188)
1614
	at org.sonar.core.platform.SpringComponentContainer.execute(SpringComponentContainer.java:167)
1615
	at org.sonar.scanner.bootstrap.SpringGlobalContainer.doAfterStart(SpringGlobalContainer.java:137)
1616
	at org.sonar.core.platform.SpringComponentContainer.startComponents(SpringComponentContainer.java:188)
1617
	at org.sonar.core.platform.SpringComponentContainer.execute(SpringComponentContainer.java:167)
1618
	at org.sonar.batch.bootstrapper.Batch.doExecute(Batch.java:72)
1619
	at org.sonar.batch.bootstrapper.Batch.execute(Batch.java:66)
1620
	at org.sonarsource.scanner.api.internal.batch.BatchIsolatedLauncher.execute(BatchIsolatedLauncher.java:46)
1621
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
1622
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
1623
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
1624
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
1625
	at org.sonarsource.scanner.api.internal.IsolatedLauncherProxy.invoke(IsolatedLauncherProxy.java:60)
1626
	at com.sun.proxy.$Proxy0.execute(Unknown Source)
1627
	at org.sonarsource.scanner.api.EmbeddedScanner.doExecute(EmbeddedScanner.java:189)
1628
	at org.sonarsource.scanner.api.EmbeddedScanner.execute(EmbeddedScanner.java:138)
1629
	at org.sonarsource.scanner.cli.Main.execute(Main.java:112)
1630
	at org.sonarsource.scanner.cli.Main.execute(Main.java:75)
1631
	at org.sonarsource.scanner.cli.Main.main(Main.java:61)
1632
Caused by: java.io.IOException: Cannot run program "node": error=2, No such file or directory
1633
	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1128)
1634
	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1071)
1635
	at org.sonarsource.nodejs.ProcessWrapperImpl.startProcess(ProcessWrapperImpl.java:39)
1636
	at org.sonarsource.nodejs.NodeCommand.start(NodeCommand.java:77)
1637
	... 38 common frames omitted
1638
Caused by: java.io.IOException: error=2, No such file or directory
1639
	at java.base/java.lang.ProcessImpl.forkAndExec(Native Method)
1640
	at java.base/java.lang.ProcessImpl.<init>(ProcessImpl.java:340)
1641
	at java.base/java.lang.ProcessImpl.start(ProcessImpl.java:271)
1642
	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1107)
1643
	... 41 common frames omitted
1644
1645
INFO: Miss the cache for 0 out of 0
1646
INFO: Sensor JavaScript analysis [javascript] (done) | time=1070ms
1647
INFO: Sensor TypeScript analysis [javascript]
1648
INFO: No input files found for analysis
1649
INFO: Hit the cache for 0 out of 0
1650
INFO: Miss the cache for 0 out of 0
1651
INFO: Sensor TypeScript analysis [javascript] (done) | time=4ms
1652
INFO: Sensor CSS Rules [javascript]
1653
INFO: Hit the cache for 0 out of 0
1654
INFO: Miss the cache for 0 out of 0
1655
INFO: Sensor CSS Rules [javascript] (done) | time=1ms

```
I guess this explained a lot

--- a/Dockerfile
+++ b/Dockerfile
@@ -28,6 +28,10 @@ RUN apt-get update -y \
     && apt-get install --no-install-recommends -y apt-transport-https \
     && apt-get update -y \
     && apt-get install --no-install-recommends -y aspnetcore-runtime-$DOTNETCORE_RUNTIME_VERSION
+    
+# Install NodeJS
+RUN curl -fsSL https://deb.nodesource.com/setup_19.x | bash - \
+    && apt-get install -y nodejs
 
 # Install Java Runtime for SonarScanner
 RUN apt-get install --no-install-recommends -y openjdk-$JRE_VERSION-jre

